components/console-backend-service/internal/domain/apigateway/apirule_converter.go

```diff
@@ -0,0 +1,127 @@
+package apigateway
+
+import (
+	"encoding/json"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kyma-incubator/api-gateway/api/v1alpha1"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/apigateway/pretty"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+	"github.com/pkg/errors"
+)
+
+type apiRuleConverter struct{}
+
+func toGQLJSON(config *runtime.RawExtension) (gqlschema.JSON, error) {
```

Comment:
I don't quite get the purpose of this function. gqlschema.JSON is already a map[string]interface{}
result.UnmarshalGQL(string(config.Raw)) should do the trick?